addons/token-exchange/secret_exchange_handler_utils.go

```diff
@@ -97,6 +100,7 @@ func generateBlueSecretForExternal(rookCephMon *corev1.Secret, labelType utils.S
 			Namespace: managedClusterName,
 			Labels: map[string]string{
 				utils.SecretLabelTypeKey: string(labelType),
+				HubRecoveryLabel:         "",
```

Comment:
is it not
HubRecoveryLabel:   "resource" ?

i can see it is used like that: https://github.com/red-hat-storage/odf-multicluster-orchestrator/pull/194/files#diff-71a31664c2a326d791c6040801a670cc01383300df3e4378f843bb619d2ea4e8R163

Reply:
https://github.com/stolostron/cluster-backup-operator